packages/web/src/components/card/Card.tsx

```diff
@@ -1,15 +1,24 @@
-import { ReactNode, Ref, forwardRef } from 'react'
+import { ReactNode, Ref, createContext, forwardRef, useContext } from 'react'
 
-import { Divider, Flex, Paper, PaperProps } from '@audius/harmony'
+import { Divider, Flex, FlexProps, Paper, PaperProps } from '@audius/harmony'
 
-type CardSize = 's' | 'm' | 'l'
+type CardSize = 'xs' | 's' | 'm' | 'l'
+
+type CardContextType = {
+  size: CardSize
+}
+
+const CardContext = createContext<CardContextType>({ size: 'm' })
 
 const cardSizes = {
+  xs: undefined,
   s: 200,
   m: 224,
   l: 320
 }
 
+const xsSize = { minWidth: 140, maxWidth: 190 }
```

Comment:
interesting. This one is meant to vary to fit its container?

Reply:
yeah exactly, cause for small devices, we want the cards to resize down to 140 so two can be side-by-side. otherwise iphone SE it's one card per line. And if we went with 140 to fix that, then larger iphones have cards with lots of space in between.